log4j-core/src/test/java/org/apache/logging/log4j/core/async/perftest/PerfTestDriver.java

```diff
@@ -192,14 +192,14 @@ public String toString() {
         }
     }
 
-    static enum Runner {
+    enum Runner {
         Log4j12(RunLog4j1.class), //
         Log4j2(RunLog4j2.class), //
         Logback(RunLogback.class);
 
         private final Class<? extends IPerfTestRunner> implementationClass;
 
-        private Runner(final Class<? extends IPerfTestRunner> cls) {
+        Runner(final Class<? extends IPerfTestRunner> cls) {
```

Comment:
Why not `private`?

Reply:
Same case above. In my opinion it's redundant.